package.json

```diff
@@ -1,12 +1,25 @@
 {
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
```

Comment:
There is a file called `packages/sample-app/src/App.tsx`, so the `.tsx` extension should also be added to this list